okapi-core/src/main/java/org/folio/okapi/managers/TenantManager.java

```diff
@@ -555,7 +556,7 @@ private void fireTimer(Tenant tenant, ModuleDescriptor md, RoutingEntry re, Stri
       pl = new PermissionList(moduleTo, mdTo.getExpandedPermissionSets());
     }
     String pljson = Json.encodePrettily(pl);
-    pc.debug("tenantPerms Req: " + pljson);
+    logger.debug("tenantPerms Req: {}", () -> pljson);
```

Comment:
Would this work `logger.debug("tenantPerms Req: {}", pljson);`?

Reply:
yes. Since it's a simple string - probably no need to.